tests/test_backend.py

```diff
@@ -1218,3 +1205,20 @@ def test_delete_pattern_calls_delete_for_given_keys(
         client.delete_pattern(pattern="foo*")
 
         connection.delete.assert_called_once_with(*connection.scan_iter.return_value)
+
+
+class TestClearClient(unittest.TestCase):
+    cache_name = "default"
```

Comment:
I see no need on putting class variables that are used only once

Reply:
I can do it, but if we need to run the same with different cache, a different key or a different value, it can be easily done by extending this class and overriding the class attributes. Than enables me to easily reuse this test here: https://github.com/jazzband/django-redis/blob/e109e4e3ce655ae4d4217150997915a43b941a55/tests/test_backend.py#L1233

Reply:
It is not readable the same way, if you subclass a test class it is not straightforward to understand that the test is ran twice with different parameters, readability is more important than writing less code. Same goes with class variables, a test should be self explanatory without knowing all the customizations of the class. These two classes I think can go in `DjangoRedisCacheTests` with two different methods